styles/mandar.css

```diff
@@ -1629,6 +1630,7 @@ img {
   line-height: auto;
   margin: 0;
   border: 0 none;
+  flex-direction: column;

```

Comment:
Won't this break v1 as well as fixing v2?

Reply:
Yes this was in the wrong spot, should properly be in .starwarsffg.v2 .tabs

Reply:
Sheet 1 working: 

![image](https://user-images.githubusercontent.com/81570172/149167577-d378559e-12db-48ec-9c63-97d06b522eca.png)

Reply:
sheet2:

![image](https://user-images.githubusercontent.com/81570172/149167695-33a27e98-e36d-4881-89cc-6682b0bdbe79.png)